### PR TITLE
[93] ensure successful building and testing for release (4-2-stable)

### PR DESCRIPTION
**Pull request by d-w-moore**
These changes were relevant to the iRODS 4.2.11 release.

* All OSes:
  - glob pattern matching working for new package names

* Centos:
  - update ca-certificates package so that RENCI repo is recognized
  - eliminate shared system dirs from RPM package

We also found CMake's demands on libssl version (=libssl.so.1.1)
are not satisfied by Ubuntu 16.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -243,4 +243,15 @@ elseif (IRODS_LINUX_DISTRIBUTION_NAME STREQUAL "opensuse")
...
+list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}etc/irods")
+list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}var")
+list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}var/lib")
+list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}etc/init.d")
+list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}${IRODS_HOME_DIRECTORY}")
+list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}${IRODS_HOME_DIRECTORY}/scripts")
+list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}${IRODS_HOME_DIRECTORY}/scripts/irods")
+list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}${IRODS_HOME_DIRECTORY}/scripts/irods/test")
+
+
```
Comment (korydraughn) [inline CMakeLists.txt:256]: unnecessary empty line

**Review by korydraughn — COMMENTED** [review]
Other than the whitespace, this looks good.

File: CMakeLists.txt (hunk 1 above)
Comment (d-w-moore) [reply CMakeLists.txt]: Good deal, thanks ... took care of it.

**Comment by trel** [comment]
yes, please # it.

**Comment by d-w-moore** [comment]
Ok, done.

**Comment by trel** [comment]
and please cherry-pick to master

**Comment by d-w-moore** [comment]
Will do.

**Pull request merged**